api/Dockerfile

```diff
@@ -38,6 +38,8 @@ WORKDIR /app
 
 COPY package*.json ./
 
+COPY add-extra-deps.js ./
```

Comment:
To be added to the template Dockerfile as well: 
`COPY --from=base /app/add-extra-deps.js ./`